### PR TITLE
i18n(fr): update `guides/astro-db.mdx`

### PR DESCRIPTION
**Pull request by ArmandPhilippot**
#### Description (required)

* Adds changes from #9509 and #9701 to the French translation of `guides/astro-db.mdx`
* Rephrases a heading in `guides/integrations-guide/db.mdx` (in French we use `table` and not `tableau` in the context of database)

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/astro-db.mdx [hunk 1]
```diff
@@ -55,11 +52,11 @@ export default defineDb({
...
 
-Lorsque vous définissez une table, Astro génère une interface TypeScript pour interroger cette table à partir de votre projet. Le résultat est un support TypeScript complet lorsque vous accédez à vos données avec l'autocomplétion des propriétés et la vérification des types.
+Définissez vos tables dans votre fichier `db/config.ts` en fournissant la structure des données de votre base de données libSQL existante ou les données que vous collecterez dans une nouvelle base de données. Cela permettra à Astro de générer une interface TypeScript pour interroger cette table à partir de votre projet. Le résultat est une prise en charge complète de TypeScript lorsque vous accédez à vos données avec la saisie semi-automatique des propriétés et la vérification des types.
 
-Pour configurer une table de base de données individuelle, importez et utilisez les utilitaires `defineTable()` et `column` de `astro:db`.
+Pour configurer une table de base de données, importez et utilisez les utilitaires `defineTable()` et `column` depuis `astro:db`. Ensuite, définissez un nom (sensible à la casse) pour votre table et le type de données dans chaque colonne.
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/astro-db.mdx:57]:
```suggestion
Pour configurer une table de base de données, importez et utilisez les utilitaires `defineTable()` et `column` depuis `astro:db`. Ensuite, définissez un nom (sensible à la case) pour votre table et le type de données dans chaque colonne.
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Good PR! Seems good, only one typo error

**Review by thomasbnt — APPROVED** [review]

File: src/content/docs/fr/guides/astro-db.mdx (hunk 1 above)
Comment (ArmandPhilippot) [reply src/content/docs/fr/guides/astro-db.mdx:57]: English influence 😅

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**